code/assurance-testapp/build.gradle

```diff
@@ -42,22 +44,48 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    kotlinOptions {
+        jvmTarget = kotlinJvmTarget
+    }
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.3.2'
+    }
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:${rootProject.ext.kotlinVersion}"
-    //noinspection GradleCompatible
-    implementation 'androidx.appcompat:appcompat:1.0.0'
+    implementation 'androidx.appcompat:appcompat:1.6.0'
+    implementation 'androidx.core:core-ktx:1.9.0'
+    implementation "androidx.compose.ui:ui:1.3.0"
+    implementation "androidx.compose.material:material:1.3.0"
+    implementation "androidx.compose.ui:ui-tooling-preview:1.3.0"
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0'
+    implementation 'androidx.activity:activity-compose:1.6.1'
+    implementation 'androidx.navigation:navigation-compose:2.5.3'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4'
+    implementation "androidx.lifecycle:lifecycle-viewmodel-compose:2.5.1"
 
-    //implementation 'com.adobe.marketing.mobile:assurance:2.0.0'
+    // implementation 'com.adobe.marketing.mobile:assurance:2.0.0'
     implementation project(':assurance')
-    implementation 'com.github.adobe.aepsdk-core-android:core:dev-v2.0.0-SNAPSHOT'
+    implementation 'com.adobe.marketing.mobile:core:2.0.0-SNAPSHOT'
+    implementation("com.adobe.marketing.mobile:signal:2.0.0-SNAPSHOT") {
+        transitive = false
+    }
+    implementation("com.adobe.marketing.mobile:places:2.0.0-SNAPSHOT") {
```

Comment:
Any reason we are including places? Does the sample have places specific use case?

Reply:
Yea, the there exists an assurance plugin for Places and a corresponding test case to simulate and load POIs and entry and exit positions. Pretty much everything is done on the Places extension side and the Assurance UI plugin. Just registering places is sufficient from Assurance sdk.